src/components/filter/expert/expert-filter-constants.ts

```diff
@@ -860,4 +899,27 @@ export const fields: Record<string, Field[]> = {
         FIELDS_OPTIONS.SUBSTATION_PROPERTY,
         FIELDS_OPTIONS.VOLTAGE_LEVEL_PROPERTY,
     ],
+    HVDC_LINE: [
```

Comment:
I don't see declaration for NOMINAL_VOLTAGE in the fields
see https://github.com/gridsuite/filter/pull/41/files#diff-a6fa1d8bbf07e8485a7dc4932ca9a10bb0ef0d7af1e0afcbef77f88b542c4fb6R72

But i don't think we can reuse NOMINAL_VOLTAGE because this is voltage of the line so it should be another label